src/extensions/secretsUtils.ts

```diff
@@ -34,6 +36,29 @@ export function getActiveSecrets(instance: extensionsApi.ExtensionInstance): str
     .filter((pv) => !!pv);
 }
 
+export async function getManagedSecrets(
```

Comment:
Also, is getActiveSecrets used anywhere now? if not, lets get rid of it to avoid bloating this code.

Reply:
getManagedSecrets still uses it, but I'm not exporting it anymore.